.github/workflows/gh-pages.yml

```
@@ -32,17 +29,21 @@ jobs:
+        run: |
+          yarn test
+
+      - name: Build component
         run: |
           yarn build
 
-      - name: Unit tests
+      - name: Build website
         run: |
-          yarn test
+          yarn build-website  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)